unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -429,18 +428,18 @@ protected void HideAllAgentRenderers()
             return reachablePos;
         }
 
-        public void GetReachablePositions(int maxStepCount = 0) {
-            if(maxStepCount != 0) {
-                reachablePositions = getReachablePositions(1.0f, maxStepCount);
+        public void GetReachablePositions(int? maxStepCount = null) {
+            Vector3[] reachablePositions;
+            if (maxStepCount != null) {
```

Comment:
When using nullable types, I prefer doing something like `maxStepCount.HasValue` rather than`maxStepCount != null` as really a nullable value is never really "equal" to null, it just fails to have a value.

Reply:
Fixed in https://github.com/allenai/ai2thor/pull/628/commits/f4f78b083cfbda37c743642548cacc5c72606492.